packages/react/tsconfig.json

```diff
@@ -19,7 +19,8 @@
     "resolveJsonModule": true,
     "skipLibCheck": true,
     "strict": true,
-    "target": "es2020"
+    "target": "es2020",
+    "typeRoots": ["node_modules/@types", "src/**/*.d.ts"]
```

Comment:
Is this necessary in the current setup?

Reply:
No, it was forgotten after testing with loading svg files.